types/jquery-toast-plugin/index.d.ts

```diff
@@ -13,19 +14,26 @@ interface JQueryStatic {
 interface toastOptions {
     text: string;
     heading?: string;
-    showHideTransition?: string;
+    showHideTransition?: 'fade' | 'slide' | 'plain';
     allowToastClose?: boolean;
-    hideAfter?: number;
+    hideAfter?: number | false;
     loader?: boolean;
     loaderBg?: string;
-    stack?: number;
-    position?: string;
-    bgColor?: boolean;
-    textColor?: boolean;
-    textAlign?: string;
-    icon?: boolean;
+    stack?: number | false;
+    position?: 'bottom-left' | 'bottom-right' | 'bottom-center' | 'top-right' | 'top-left' | 'top-center' | 'mid-center' | CustomPosition;
+    bgColor?: string;
+    textColor?: string;
+    textAlign?: 'left' | 'right' | 'center';
+    icon?: 'info' | 'warning' | 'error' | 'success';
     beforeShow?: () => any;
     afterShown?: () => any;
     beforeHide?: () => any;
     afterHidden?: () => any;
 }
+
+interface CustomPosition {
+    left: number | 'auto';
```

Comment:
Documentation seems to use `'-'` as an alternative to `'auto'`: https://kamranahmed.info/toast#toasts-positioning

Any idea if you should include that?